src/components/form/form.jsx

```diff
@@ -260,7 +260,7 @@ Form.Combobox = ({
   setValue,
   ...rest
 }) => {
-  const selectedItems = watch(id) || [];
```

Comment:
Was this change needed?

Reply:
I'm can't remember. I can take a look if you want, but I expect @dani9401 's combo box PR to change this code regardless, as it deals with this area directly.